lib/tag-generator.js

```diff
@@ -10,20 +11,16 @@ export default class TagGenerator {
   }
 
   getPackageRoot() {
-    let packageRoot
-    if (typeof snapshotResult === 'undefined') {
-      packageRoot = path.resolve(__dirname, '..');
+    if (fs.isAbsolute(__dirname)) {
+      return path.resolve(__dirname, '..')
```

Comment:
Thanks for the heads-up, @maxbrunsfeld. I thought I had tested it on the current Atom version but it looks like I overlooked something. I will work on a fix shortly!